test/quantization/test_quantized_tensor.py

```diff
@@ -257,6 +307,41 @@ def quantize_ref(data, scales, zero_points):
         self.assertTrue(np.allclose(qr.int_repr(), ref))
         self.assertTrue(np.allclose(r.numpy(), dequant_tensor.numpy(), atol=1))
 
+        def _quantize_per_channel_ref_nd(data, scales, zero_points):
```

Comment:
this function looks very similar to the one in test_qtensor_quantize_per_channel, can we merge them

Reply:
I did that. But there was a reason why I reverted that. Now it is escaping me. I will try to merge, then it will come back.

Reply:
OK, so now I remember. In one of them rounding happens before while the other rounding happens after adding zero point. So could probably add a parameter to the function to chose one over the other, but I opted for just keeping two copies.

Reply:
Ok done.